jobs/JGLOBAL_MARINE_ANALYSIS_LETKF

```diff
@@ -11,12 +18,19 @@ GDATE=$(date --utc +%Y%m%d%H -d "${PDY} ${cyc} - ${assim_freq} hours")
 
 gPDY=${GDATE:0:8}
 gcyc=${GDATE:8:2}
+export GDUMP="gdas"
+export GDUMP_ENS="enkf${GDUMP}"
+export RUN="${GDUMP}"
```

Comment:
@WalterKolczynski-NOAA I don't think `RUN` can be assigned here. It is set at the rocoto level, correct?

Reply:
changeed to walter' suggestion